### PR TITLE
Align Aldec simulators with others based on #1063

### PR DESCRIPTION
**Pull request by themperek**
Align based on the discussion in #1063

It is failing at the moment because of the wrong locations of `results.xml` and https://github.com/cocotb/cocotb/pull/1595

Other simulators will follow ...

### CODE REVIEW

File: cocotb/share/makefiles/simulators/Makefile.aldec [hunk 1]
```diff
@@ -63,8 +63,8 @@ ASIM_ARGS += $(SIM_ARGS)
 # Plusargs need to be passed to ASIM command not vsimsa
 ASIM_ARGS += $(PLUSARGS)
 
-RTL_LIBRARY ?= work
-ALOG_ARGS += +define+COCOTB_SIM -dbg -pli libgpi
+RTL_LIBRARY ?= $(SIM_BUILD)/work
+ALOG_ARGS += +define+COCOTB_SIM -dbg
```
Comment (eric-wieser) [inline cocotb/share/makefiles/simulators/Makefile.aldec:67]: Removing `-pli libgpi` seems out of scope for this patch. Was it necessary? Can you put it in a separate commit?

File: .gitignore [hunk 2]
```diff
@@ -80,3 +80,12 @@ tests/test_cases/*/transcript
...
 *.wlf
+
+# Riviera
+tests/test_cases/*/library.cfg
+tests/test_cases/*/dataset.asdb
+tests/test_cases/*/compile
+examples/*/tests/library.cfg
+examples/*/tests/dataset.asdb
+examples/*/tests/compile
+examples/*/tests/*.mgf
```
Comment (eric-wieser) [inline .gitignore:91]: Any reason this one is here but not in the `clean` or `test_cases` paths?

Comment (themperek) [reply .gitignore]: Legacy. Sorry.

**Comment by themperek** [comment]
🤞

**Comment by eric-wieser** [comment]
@themperek: Based on #1061, this didn't help (although appears not to have made things worse)
